go.mod

```diff
@@ -1,10 +1,9 @@
 module openebs.io/metac
 
-go 1.13.5
+go 1.13
```

Comment:
This denotes the minimum go language supported version and fails with go `1.14` if the minor version is included:
```
go: errors parsing go.mod:
/workingCopies/src/github.com/AmitKumarDas/metac/go.mod:3: usage: go 1.23
make: *** [Makefile:67: vendor] Error 1
```

Reply:
Great. Thanks for the info. This is a new info to me.
We might want to put up a comment there.

Reply:
sure, I've added a comment.